pkg/eden/qemu.go

```diff
@@ -193,6 +225,9 @@ func StartEVEQemu(qemuARCH, qemuOS, eveImageFile, imageFormat string, isInstalle
 		qemuOptions += fmt.Sprintf("-readconfig %s ", qemuConfigFile)
 	}
 
+	// QMP sock
+	qemuOptions += fmt.Sprintf("-qmp unix:%s,server,wait=off", qmpSock)
```

Comment:
And if we can't start this thing, qmpSock will be file that does not exist, will qemu just send warning message or will it crash in that case?

Reply:
QEMU creates a socket file and uses is as server socket. socat attaches to it, sends handshake message and redirects output to a file. if socket file was not created for 5 seconds, the `startQMPLogger()` complains, but this error is not considered as fatal.